tool/site/server_hapi/src/routes/prescribe/send.ts

```diff
@@ -45,15 +50,17 @@ export default [
         const sendRequest = getSessionValue(`prescription_order_send_request_${prescriptionIds[0]}`, request)
         const sendResponse = await epsClient.makeSendRequest(sendRequest)
         const sendRequestHl7 = await epsClient.makeConvertRequest(sendRequest)
-        return responseToolkit.response({
+        const sendResult = {
           prescription_ids: prescriptionIds,
           prescription_id: prescriptionIds[0],
           success: true,
           request_xml: sendRequestHl7,
           request: sendRequest,
           response: sendResponse.fhirResponse,
           response_xml: sendResponse.spineResponse
-        }).code(200)
+        }
+        setSessionValue(`signature_token_${signatureToken}`, sendResult, request)
```

Comment:
This key seems a bit odd (we're not storing a signature token here). Session storage as a whole needs some thought, so not overly worried about changing it right now